app/assets/stylesheets/govuk-component/_related-items.scss

```diff
@@ -0,0 +1,32 @@
+.govuk-related-items {
+  border-top: 10px solid $mainstream-brand;
+  padding-top: 5px;
+
+  h2 {
+    @include core-24;
+    font-weight: 600;
```

Comment:
Good spot – we should update this. It has been ported from: https://github.com/alphagov/static/blob/master/app/assets/stylesheets/helpers/_core.scss#L64